apps/frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -24,10 +27,15 @@ const Sidebar: React.FC<Properties> = ({
 
 	const dispatch = useAppDispatch();
 
-	const handleSignOut = useCallback(
-		() => void dispatch(authActions.logOut()),
-		[dispatch],
-	);
+	const [isLogout, setIsLogout] = useState<boolean>(false);
```

Comment:
```suggestion
	const [isLogoutPopupOpen, setIsLogoutPopupOpen] = useState<boolean>(false);
```

Reply:
I feel like this logic shouldn't be in sidebar but in either App.tsx or ProtectedRoute.tsx, or even Settings.tsx since we'll move the sign out button to settings.

Reply:
We'll move it to Settings, when we will move the sign out button there